oomact_ros/src/feeder/ImuInputFeeder.cpp

```diff
@@ -0,0 +1,46 @@
+#include <aslam/calibration/data/ObservationManagerI.h>
+#include <aslam/calibration/data/MeasurementsContainer.h>
+#include <aslam/calibration/data/AccelerometerMeasurement.h>
+#include <aslam/calibration/data/GyroscopeMeasurement.h>
+#include <aslam/calibration/input/InputReceiverI.h>
+#include <rosbag/bag.h>
+#include <rosbag/view.h>
+#include <sensor_msgs/Imu.h>
+
+#include "aslam/calibration/ros/InputFeederFactoryRegistry.h"
+#include "aslam/calibration/ros/InputFeederFactoryI.h"
+#include "aslam/calibration/ros/InputFeederTemplates.h"
+
+namespace aslam {
+namespace calibration {
+namespace ros {
+
+bool msg2Measurement(const sensor_msgs::Imu &msg, AccelerometerMeasurement & m){
+  if(msg.linear_acceleration_covariance[0] == 0.0){
```

Comment:
From the Imu message def http://docs.ros.org/api/sensor_msgs/html/msg/Imu.html
If an acceleration isn't present the first value is set to -1, so it probably makes more sense to check if value <= 0.0

Reply:
Ah, good catch! I've remembered it wrongly. I though -1 is no covariance available and 0 is skipping. But it is flipped. Here I only need to test for skipped. The no covariance available is handled in the Imu sensor model.